python/ambassador/compile.py

```diff
@@ -70,7 +67,6 @@ def Compile(logger: logging.Logger, input_text: str,
     out: Dict[str, Union[IR, EnvoyConfig]] = { "ir": ir }
 
     if ir:
-        out[envoy_version.lower()] = EnvoyConfig.generate(ir, envoy_version.upper(), cache=cache)
+        out['xds'] = EnvoyConfig.generate(ir, cache=cache)
```

Comment:
same comment as above...i'm not against this but it does mean that if envoy introduces a v4 then we would be back to including a version. So, I don't see a downside to leaving this